tests/manifests/special-cases/no-strategy/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ resources:
   - ../../default
 
 patches:
-  - path: usde-config.yaml
\ No newline at end of file
+  - path: controller-config.yaml
```

Comment:
no need to override `usde-config.yaml` anymore as it's identical to the default one (this was actually the main motivation for [this](https://github.com/numaproj/numaplane/pull/416) change)